src/control-plane/backend/click-stream-api.ts

```diff
@@ -295,6 +295,7 @@ export class ClickStreamApiConstruct extends Construct {
         STS_UPLOAD_ROLE_ARN: uploadRole.roleArn,
         API_ROLE_NAME: clickStreamApiFunctionRole.roleName,
         HEALTH_CHECK_PATH: props.healthCheckPath,
+        QUICKSIGHT_CONTROL_PLANE_REGION: props.targetToCNRegions ? 'cn-north-1' : 'us-east-1',
```

Comment:
QuickSight is unavailable in China regions now, how do we know the region for its control plane?

Reply:
We don't know, but the SDK client need set a default value.